### DISCUSSION TITLE
Location based search parameters for v2 API

### DISCUSSION THREAD

**Discussion by KronikerDelta** [tweepy/tweepy#1795] [Questions] [answered]
Hi, 
I've been using the `geocode` parameter in `api.search()` with the v1 API and it has been very helpful, but I don't see that functionality in your v2 API implementation in version 4.5, did twitter remove that functionality or is it encoded elsewhere in tweepy for the v2 API?

**Comment by Harmon758** [comment, marked as answer]
[The Twitter API v2 search endpoints allow for an equivalent `point_radius` operator in the query](https://developer.twitter.com/en/docs/twitter-api/tweets/search/integrate/build-a-query).
